client/src/helpers/sortObjectKeys.js

```diff
@@ -0,0 +1,20 @@
+// transforms an object into an array
+
+export const sortedObjectKeys = (obj, sort) => {
```

Comment:
maybe add a note that explains this is alphabetical order since it isn't explicit